internal/storage/storageutil/client_test.go

```diff
@@ -55,7 +81,7 @@ func (t *clientTest) TestCreateTokenSrcWhenCustomEndpointIsNil() {
 
 func (t *clientTest) TestCreateHttpClientWithHttp1() {
 	sc := GetDefaultStorageClientConfig() // By default http1 enabled
-
+	sc.DisableAuth = true
```

Comment:
TestCreateHttpClientWithHttp1 doesn't say about DisableAuth. So, we shouldn't set DisableAuth inside.

Should we write tests for all possible combinations ((http1 and http2) * (disable auth and enable auth)) ?